app/models/task.rb

```diff
@@ -2,4 +2,17 @@
 
 class Task < ActiveRecord::Base
   validates :name, presence: true
+  validates :completed, on: :create, acceptance: { accept: false, message: 'Task cannot be marked as completed when created.' }
+
+  def self.get_list(completed_param)
```

Comment:
I find this param name a little confusing, since the the param isn't necessarily `== completed`. 

Also, wondering if there is a way to use a model scope/active record to return an active record collection instead of initially setting an array and then having to return it at the end of the class method.

Reply:
I think it would be nice to add a model spec for this method.

Reply:
Used scope to return completed and incompleted tasks and changed <code>get_list(completed_param)</code> to <code>get_tasks(task_type)</code>. Added a model spec for this method.